lib/util/mag_color.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:flutter/material.dart';
+
+extension MagnitudeColorExtension on double {
```

Comment:
```suggestion
extension MagnitudeColorExtension on ColorScheme {
```
這邊是在處理顏色，擴充 `ColorScheme` 才對
在 double 上擴充會很不合理